ci/run-tests.ps1

```diff
@@ -37,12 +39,10 @@ Write-Log "Running $($ue_path_absolute) $($cmd_args_list)"
 $run_tests_proc = Start-Process -PassThru -NoNewWindow $ue_path_absolute -ArgumentList $cmd_args_list
 Wait-Process -Id (Get-Process -InputObject $run_tests_proc).id
 
-# Workaround for UNR-2156, where spatiald / runtime processes sometimes never close
+# Workaround for UNR-2156 and UNR-2076, where spatiald / runtime processes sometimes never close, or where runtimes are orphaned
```

Comment:
By the way, I think @Joe-Improbable implemented a fix for this in the GDK. Maybe we can rely on it, but I would rather leave the stop-process in, just in case.

Reply:
Looking at the PR, it's a prompt so it doesn't apply in this case (https://github.com/spatialos/UnrealGDK/pull/1416)